README.md

```diff
@@ -22,25 +22,25 @@ This is a sample NextJS webapp that uses StatelyDB.
 ## Quick Start
 
 1. Clone this repository
-2. Install dependencies:
+2. Install NPM dependencies:
    ```
    npm install
    ```
 3. Visit the Stately Cloud console at https://console.stately.cloud and grab your credentials and Store ID. If you don't have one yet, request one.
 4. Create a new Netlify site:
    ```
-   netlify sites:create
+   npx netlify sites:create
```

Comment:
Added a dependency on netlify-cli and using npx so users don't have to separately install netlify.